app/[...slug]/metadata.tsx

```diff
@@ -35,7 +35,7 @@ export const getNodeMetadata = (node: NodeUnion): Record<string, any> => {
   return {
     ...metadata,
     metadataBase: new URL("https://library.stanford.edu"),
-    title: node.title + " | " + process.env.NEXT_PUBLIC_SITE_NAME,
+    title: node.title + " | " + `Stanford ${process.env.NEXT_PUBLIC_SITE_NAME}`,
```

Comment:
we can probably just hard code the whole thing to `"Stanford Libraries"`

Reply:
Sounds good to me 😎